.github/workflows/release.yml

```diff
@@ -2,12 +2,13 @@ on:
   push:
     tags:
       - "v*" # push events to matching v*, i.e. v1.0, v20.15.10
+  workflow_dispatch:
```

Comment:
What is this needed for?

Reply:
Workflow dispatch allows collaborators (like you) to manually run the workflow. Now that I go through again it is already set to run on tags. So this won't be necessary, so I will remove it.

```suggestion
```